generic/tests/netperf.py

```diff
@@ -136,8 +144,24 @@ def mtu_set(mtu):
                         "The ethenet device does not support jumbo," "cancel test"
                     )
 
-    vm = env.get_vm(params["main_vm"])
-    vm.verify_alive()
+    pinned_node = 0
+    host_numa = utils_misc.NumaInfo()
+    node_list = host_numa.online_nodes_withcpumem
+    try:
+        for node in node_list:
+            node_mem_free = int(host_numa.read_from_node_meminfo(node, "MemFree"))
+            mem = int(params["mem"])
+            mem_kb = mem * 1024
```

Comment:
Move it outside loop, don't need to calculate the memory size for each loop

Reply:
Code updated, thank you !